src/mito2/src/metrics.rs

```diff
@@ -316,4 +323,12 @@ lazy_static! {
         "mito manifest operation elapsed",
         &["op"]
     ).unwrap();
+
+    // Parquet related metrics:
+
+    /// Elapsed time of loading parquet metadata.
+    pub static ref PARQUET_METADATA_LOAD_ELAPSED: Histogram = register_histogram!(
+        "greptime_parquet_metadata_load_elapsed",
+        "parquet metadata load elapsed",
+    ).unwrap();
```

Comment:
You can reuse the `READ_STAGE_ELAPSED` metric and add a stage `load_parquet_meta`/`load_parquet_metadata`.

Reply:
I think it's good to have some metrics all related to parquet read, this can be their first one, doesn't have to be a stage of the query.

Reply:
> I think it's good to have some metrics all related to parquet read

Then we should add a histogram vec instead of a dedicated metric for each operation. Maintaining too many metrics is a burden. You might need to modify the dashboard panel every time we add a new metric.

We also have to set the bucket size of the histogram as the default bucket size is not suitable.